### PR TITLE
delete the briefcase .bim file on a failed download

### PR DESCRIPTION
**Pull request by jiaruiz717**
fix iTwin/itwinjs-backlog#1114

Delete the .bim briefcase on a failed download

### CODE REVIEW

**Review by pmconne — APPROVED** [review]
`rush change`.

File: core/backend/src/BriefcaseManager.ts [hunk 1]
```diff
@@ -219,7 +219,14 @@ export class BriefcaseManager {
     } catch (error: unknown) {
       if (arg.accessToken && arg.briefcaseId === undefined)
         await this.releaseBriefcase(arg.accessToken, { briefcaseId, iModelId: arg.iModelId });
-
+      if (IModelJsFs.existsSync(fileName)) {
+        try {
+          IModelJsFs.unlinkSync(fileName);
+          Logger.logInfo(loggerCategory, `Deleted ${fileName}`);
```
Comment (pmconne) [inline core/backend/src/BriefcaseManager.ts:225]: Consider amending your log messages to indicate why you tried to delete the briefcase.

Comment (jiaruiz717) [reply core/backend/src/BriefcaseManager.ts:226]: Resolved in [commit 3a4d3784b2d4b92344abb4026ed8d9cb08ad6db2](https://github.com/iTwin/itwinjs-core/pull/6805/commits/3a4d3784b2d4b92344abb4026ed8d9cb08ad6db2)

File: common/changes/@itwin/core-backend/jiarui-deleteBriefcaseDbFileOnAFailedDownload_2024-06-06-20-04.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "delete breifcase .bim file on a failed download",
```
Comment (aruniverse) [inline common/changes/@itwin/core-backend/jiarui-deleteBriefcaseDbFileOnAFailedDownload_2024-06-06-20-04.json:5]:
```suggestion
      "comment": "delete briefcase .bim file on a failed download",
```

**Pull request merged**